src/tutorials/dropbox.md

```diff
@@ -0,0 +1,72 @@
+# Setting up the Dropbox daemon
+
+- Time needed: ~10 minutes
+
+Dropbox has a very useful [Linux version](https://www.dropbox.com/install-linux) that can be installed into your Platform.sh application with a bit of intermediate [build hook](/configuration/app-containers.html#hooks) hacking.
+
```

Comment:
The introduction should include when/why you'd want to do this. It's not something most people are likely to think of.